### PR TITLE
Fix bug in GitUtility

### PR DESCRIPTION
**Pull request by qinezh**

### CODE REVIEW

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Common/Git/GitUtility.cs [hunk 1]
```diff
@@ -60,7 +60,15 @@ public static GitRepoInfo GetRepoInfo(string directory)
         #region Private Methods
         private static GitDetail GetFileDetailCore(string filePath)
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Common/Git/GitUtility.cs]: >filePath [](start = 58, length = 8)

is it possible that filePath is a relative path and GetDirectoryName returns empty?

Comment (qinezh) [inline src/Microsoft.DocAsCode.Common/Git/GitUtility.cs]: nice catch, `GitUtility` should handle the case.

---
In reply to: [91219667](https://github.com/dotnet/docfx/pull/969#discussion_r91219667) [](ancestors = 91219667)

**Pull request merged**